rust-runtime/aws-smithy-types/src/lib.rs

```diff
@@ -13,16 +13,17 @@
     unreachable_pub
 )]
 
-use crate::error::{TryFromNumberError, TryFromNumberErrorKind};
 use std::collections::HashMap;
 
 pub mod base64;
 pub mod date_time;
 pub mod endpoint;
 pub mod error;
+mod number;
 pub mod primitive;
 pub mod retry;
 pub mod timeout;
+pub use number::Number;
```

Comment:
Can you move this `use` statement down with the other `use` statements?

Reply:
https://github.com/awslabs/smithy-rs/pull/2197

I made a draft PR for adding `rustfmt.toml`.   
`cargo fix` can automatically fix up things like the one you've mentioned and I think these things should be automated.

Can you check my PR?

Reply:
Great idea, I'll check it out.